Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1121,6 +1121,8 @@ Note: To enrich an incident created via the Demisto BOT (`new incident` command)
 
 ## Troubleshooting
 
+ - Note: most of the following steps do not apply to XSOAR 8, they will be adjusted in the near future.
```

Comment:
This note isn't good, as it doesn't tell what does apply and what doesn't. If they're all not supported, we should just say that - if some are supported but others are not supported - we should add a note next to each step.